components/common/autoTeleport/AutoTeleportModal.vue

```diff
@@ -0,0 +1,240 @@
+
+  if (
+    props.transactions.teleport.status.value === TransactionStatus.Finalized
+  ) {
+    if (props.canDoAction) {
+      status = TransactionStepStatus.COMPLETED
+    } else {
+      status = TransactionStepStatus.LOADING
+    }
+  }
+
+  return status
+})
+
+const steps = computed<TransactionStep[]>(() => {
+  return [
+    {
+      title: $i18n.t('autoTeleport.steps.1.title'),
+      subtitle: $i18n.t('autoTeleport.steps.1.subtitle'),
+      status: props.transactions.teleport.status.value,
+      isError: props.transactions.teleport.isError.value,
+      txId: props.transactions.teleport.txId.value,
+      prefix: props.transition.source?.prefix,
+      withAction: true,
+      retry: () => emit('telport:retry'),
+    },
+    {
+      title: $i18n.t('autoTeleport.steps.2.title'),
+      subtitle: $i18n.t('autoTeleport.steps.2.subtitle'),
+      stepStatus: checkBalanceState.value,
+    },
+    props.transactions.actions.map((action) => {
+      const { title, subtitle } = getActionDetails(action.interaction)
+      return {
+        title,
+        subtitle,
+        status: action.status,
+        isError: action.isError,
+        txId: action.txId,
+        isLoading: action.isLoading,
+        prefix: props.transition.destination?.prefix,
+        withAction: true,
+        stepStatus: undefined,
+        retry: () => emit('action:retry', action.interaction),
+      }
+    }),
+  ].flat()
+})
+
+const handleActiveStep = (step) => {
+  activeStep.value = step
+}
+
+const btnDisabled = computed(() => {
+  return !actionsFinalized.value
+})
+
+const actionsFinalized = computed(() =>
+  props.transactions.actions
+    .map((action) => action.status)
+    .every((status) => status === TransactionStatus.Finalized),
+)
+
+const btnLabel = computed(() => {
+  if (!props.canDoAction) {
+    return $i18n.t('autoTeleport.finishAllStepsFirst')
+  }
+
+  if (!actionsFinalized.value) {
+    return getActionDetails(activeStepInteraction.value).submit
+  }
+
+  return $i18n.t('autoTeleport.close')
+})
+
+const getActionDetails = (interaction: string) => {
+  const i = interaction.toLocaleLowerCase()
+  return {
+    title: $i18n.t(`autoTeleport.steps.${i}.title`),
+    subtitle: $i18n.t(`autoTeleport.steps.${i}.subtitle`),
+    submit: $i18n.t(`autoTeleport.steps.${i}.submit`),
+  }
+}
+
+const FIRST_ACTION_STEP = 2
+const activeStepInteraction = computed(
+  () =>
+    props.transactions.actions[activeStep.value - FIRST_ACTION_STEP]
+      ?.interaction,
+)
+
+watch(activeStep, () => {
+  const isActionStep = activeStep.value >= FIRST_ACTION_STEP
+  if (isActionStep) {
+    emit('action:start', activeStepInteraction.value)
+  }
+})
+
+const submit = () => {
+  if (actionsFinalized.value) {
+    onClose()
+  }
+}
+
+const onClose = () => {
+  emit('close')
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/abstracts/variables';
+
+.top {
+  z-index: 1000;
+}
+
+.modal-width {
+  width: 25rem;
+}
+
+.btn-height {
```

Comment:
I think this is recurring issue, confirm buttons are a size that is not the same as size large and also changes the button font size  to a larger one 


we might want to do something since it's spreading ...
![CleanShot 2023-11-07 at 09 46 29@2x](https://github.com/kodadot/nft-gallery/assets/44554284/36735b75-7bc7-46e5-9225-853bb146fcf1)

Reply:
oh no ... 😦 
let's open

Reply:
continue #7957